frontend/src/components/common/UserNameTooltip.tsx

```diff
@@ -38,7 +38,7 @@ export const UserNameTooltip: React.FunctionComponent<IUserNameTooltipProps> = (
   return (
     <TooltipIcon
       toolTipId={'userNameTooltip'}
-      className={'userNameTooltip'}
```

Comment:
fair amount of existing code touched by this change. I'll leave it up to you if any of these changes should be shared with Sue for revalidation.

Reply:
yeah, had to modify this component to support rich-text in tooltips. Will let Sue know